src/frontend/src/store/index.ts

```diff
@@ -2,12 +2,16 @@ import axios from "axios";
 import Vue from "vue";
 import Vuex from "vuex";
 import VueToast from 'vue-toast-notification';
+import { sha256 } from "js-sha256";
 import 'vue-toast-notification/dist/theme-sugar.css';
+import * as rs from 'jsrsasign';
 
```

Comment:
Could you add two comments surrounding the creating signature section? So write "// CREATE TRANSACTION SIGNATURE" and then right before Vue.use(Vuex) write "// END". That way it's not just random code happening before the store section, and it has an explanation for being there.

Reply:
done!